content/02.author-sections.md

```diff
@@ -234,8 +234,6 @@ More information on the tool and its inception is available in the project manus
 
 <!-- Casey Greene -->
 
-### Casey Greene
```

Comment:
@dhimmel I saw this in another PR as well.  What was the cause?

Reply:
https://github.com/dhimmel/psb-manuscript/commit/7760da8381cf48568f96ca5ad180f0c09d19f9e5 was committed after participants had clicked the github pencil icon and before they proposed changes.